src/ergw_aaa_radius.erl

```diff
@@ -870,7 +870,18 @@ send_request(Req, Session, #{server := NAS, retries := Retries, timeout := Timeo
 			{server_name, maps:get(server_name, Opts, Id)},
 			{retries, Retries},
 			{timeout, Timeout}],
-    eradius_client:send_request(NAS, Req, RadiusClientOpts).
+    case eradius_client:send_request(NAS, Req, RadiusClientOpts) of
+        {ok, _, _} = Res ->
+            Res;
+        _Err ->
+            %% otherwise NAS is map of server and/or server_backup entries
+            case eradius_client:send_request(maps:get(server_beckup, Opts, NAS), Req, RadiusClientOpts) of
```

Comment:
typo?: b**e**ckup  ( `e` instead of `a` )

Reply:
Copy-paste costs :smiley:. Good point about tests, thanks! Fixed.